src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -59,8 +62,12 @@ public HttpResponseMessage Invoke(string name, [FromBody] FunctionInvocation inv
         [Route("admin/functions/{name}/status")]
         public FunctionStatus GetFunctionStatus(string name)
         {
-            FunctionStatus status = new FunctionStatus();
             Collection<string> functionErrors = null;
+            FunctionDescriptor function = _scriptHostManager.Instance.Functions.FirstOrDefault(p => p.Name.ToLowerInvariant() == name.ToLowerInvariant());
+            FunctionStatus status = new FunctionStatus
+            {
+                Metadata = function?.Metadata
```

Comment:
I look at ScriptType to determine the language for debugging, and InputBinding for `IsHttpFunction` check when running.